environment.py

```diff
@@ -526,5 +526,12 @@ def env() -> Mapping[str, Optional[str]]:
         # testing feature branches in GitLab before they are merged to the
         # develop branch, 0 otherwise. Personal deployments have this set to 0.
         #
-        'AZUL_IS_SANDBOX': '0'
+        'AZUL_IS_SANDBOX': '0',
+
+        # The UCSC-GI organization's Slack workspace ID.
```

Comment:
As mentioned before, the point of making something configurable is to avoid having to specify, document or otherwise bake in assumptions about concrete values of that configuration.